File: src/java/eu/sig/training/ch06/userservice/splitted/UserBlockService.java
package eu.sig.training.ch06.userservice.splitted;

import java.util.ArrayList;
import java.util.List;

import eu.sig.training.ch06.userservice.User;

@SuppressWarnings("unused")
// tag::UserBlockService[]
public class UserBlockService {
    public void blockUser(User user) {
        // ... 
    }

    public List<User> getAllBlockedUsers() {
        // ...
        // end::UserBlockService[]
        return new ArrayList<User>();
        // tag::UserBlockService[]
    }
}
// end::UserBlockService[]

File: src/csharp/eu/sig/training/ch06/userservice/splitted/UserBlockService.cs
using System.Collections.Generic;

namespace eu.sig.training.ch06.userservice.splitted
{

    // tag::UserBlockService[]
    public class UserBlockService
    {
        public void BlockUser(User user)
        {
            // ...
        }

        public IList<User> GetAllBlockedUsers()
        {
            // ...
            // end::UserBlockService[]
            return new List<User>();
            // tag::UserBlockService[]
        }
    }
    // end::UserBlockService[]

}
